indico/modules/events/notifications.py

```diff
@@ -27,14 +41,47 @@ def notify_event_creation(event, occurrences=None):
              filter(Category.notify_managers | (Category.event_creation_notification_emails != []))
              .options(joinedload('acl_entries')))
     for cat in query:
-        emails.update(cat.event_creation_notification_emails)
-        if cat.notify_managers:
-            for manager in cat.get_manager_list():
-                if manager.principal_type in (PrincipalType.user, PrincipalType.email):
-                    emails.add(manager.email)
-                elif manager.principal_type in (PrincipalType.local_group, PrincipalType.multipass_group):
-                    emails.update(x.email for x in manager.get_members())
+        emails.update(_get_emails_from_category(cat))
 
     if emails:
         template = get_template_module('events/emails/event_creation.txt', event=event, occurrences=occurrences)
         send_email(make_email(bcc_list=emails, template=template))
+
+
+def notify_move_request_creation(events, target_category, comment=''):
+    """Send email notifications when a move request is created.
+
+    :param events: List of events requested to move.
+    :param target_category: The category to move the events into.
+    :param comment: Optional requestor's comment.
+    """
+    emails = set()
+    query = (target_category.chain_query
+             .filter(Category.notify_managers | (Category.event_creation_notification_emails != []))
+             .options(joinedload('acl_entries')))
+
+    # Walk up the category chain
+    for cat in reversed(query.all()):
+        emails.update(_get_emails_from_category(cat, ignore_notify_managers=True))
+
+        if emails:
+            template = get_template_module('events/emails/move_request_creation.txt',
+                                           events=events, target_category=target_category, comment=comment)
+            send_email(make_email(bcc_list=emails, template=template))
+            break
+
+
+def notify_move_request_closure(move_requests, accept, reason=''):
+    """Send email notifications when a move request is accepted/rejected.
+
+    :param move_requests: List of `EventMoveRequest` that were accepted/rejected.
+    :param accept: Whether the requests were accepted.
+    :param reason: Optional reason for rejection.
+    """
+
+    move_requests = sorted(move_requests, key=lambda rq: rq.requestor.id)
+    for requestor, requests in itertools.groupby(move_requests, lambda rq: rq.requestor):
+        events = [rq.event for rq in requests]
+        template = get_template_module('events/emails/move_request_closure.txt',
+                                       events=events, accept=accept, reason=reason)
+        send_email(make_email(bcc_list=requestor.email, template=template))
```

Comment:
There's no need to use bcc here - it goes directly to a single user -> use `to_list`